packages/app/src/App.tsx

```diff
@@ -15,14 +15,16 @@
  */
 
 import { CssBaseline, makeStyles, ThemeProvider } from '@material-ui/core';
-import { BackstageTheme, createApp } from '@backstage/core';
-import React, { FC } from 'react';
+import { BackstageThemeLight, BackstageThemeDark, createApp } from '@backstage/core';
+import React, {FC} from 'react';
 import { BrowserRouter as Router } from 'react-router-dom';
 import Root from './components/Root';
 import ErrorDisplay from './components/ErrorDisplay';
 import * as plugins from './plugins';
 import apis, { errorDialogForwarder } from './apis';
 
+import {ThemeContext, ThemeContextType, useThemeType} from './ThemeContext';
```

Comment:
Same as above, no Prettier.

Reply:
Hi, I've prettified the files now and pushed a new commit this morning, did you see it?